qmlbind/include/qmlbind/metaclass.h

```diff
@@ -6,56 +6,54 @@ extern "C" {
 #endif
 
 /*!
- * \defgroup exporter_module qmlbind_exporter
- * \brief provides a builder-style API for creating a new \ref metaobject_module .
+ * \defgroup metaclass_module qmlbind_metaclass
+ * \brief Defines an object type for use in QML type system.
  *
- * After adding all wanted methods, signals and properties, an exporter should be transformed into a \ref
- * `metaobject_module` by using `qmlbind_exporter_into_metaobject()`.
+ * It enables you to wrap client objects into a [QML object type](http://doc.qt.io/qt-5/qtqml-typesystem-objecttypes.html) and instantiate them in QML.
+ * It is also possible to create wrappers as `qmlbind_value` in `qmlbind_engine_new_wrapper`.
+ *
+ * Internally, the `qmlbind_metaclass` creates a [QMetaObject](https://doc.qt.io/qt-5/qmetaobject.html) for the wrapper Qt objects.
+ *
+ * libqmlbind's rough equivalent of [Meta-Object Compiler](http://doc.qt.io/qt-5/moc.html).
  */
 
-/** \addtogroup exporter_module
+/** \addtogroup metaclass_module
  *  @{
  */
 
-/*! \struct qmlbind_exporter
- * \brief an opaque struct mainly used as `self` argument in the methods defined in \ref exporter_module.
+/*! \struct qmlbind_metaclass
+ * \brief an opaque struct mainly used as `self` argument in the methods defined in \ref metaclass_module.
  */
 
-/*! \file exporter.h
- * \brief Contains all methods defined on `qmlbind_exporter`.
+/*! \file metaclass.h
+ * \brief Contains all methods defined on `qmlbind_metaclass`.
  */
 
 /*!
- * \brief Creates a new `qmlbind_exporter`.
+ * \brief Creates a new `qmlbind_metaclass`.
  *
  * \param classObject will be given as parameter to the `new_object` function of the provided
  * `qmlbind_client_callbacks`. You can use it to hand over the equivalent of class objects in your language to
  * implement a generic `new_object` function. If you don't need it, just hand over a nullpointer. This function does not
  * take ownership of the classObject, but it still needs to be valid for as long as new objects are created (i.e. the
  * whole application lifetime).
  *
- * \param className is used for registering the metaobject created with this exporter in the metaobject system.
+ * \param className is used for registering the metaobject created with this metaclass in the metaobject system.
```

Comment:
Because the user now registers metaclasses instead of -objects via `qmlbind_metaclass_register()`, I think it makes sense to rephrase it to something like this: `\param className is used for registering this metaclass to the metaobject system.`